### PR TITLE
[1773] Update fop downloader

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: business_register/converter/fop_full.py [hunk 1]
```diff
@@ -290,3 +298,80 @@ def save_to_db(self, records):
         self.bulk_manager.queues['business_register.ExchangeDataFop'] = []
 
     print("For storing run FopFullConverter().process()")
+
+
+class FopFullDownloader(Downloader):
+    chunk_size = 16 * 1024 * 1024
+    reg_name = 'business_fop'
```
Comment (PyDev777) [inline business_register/converter/fop_full.py:305]: business_fop_full? (business_fop used for short source file)

File: data_ocean/templates/data_ocean/emails/report.html [hunk 2]
```diff
@@ -29,7 +29,7 @@
         {% for report in reports %}
             <tr>
                 <td>{{ report.registry_name }}</td>
-                {% if report.registry_name == 'business_ukr_company' %}
+                {% if report.registry_name == 'business_ukr_company' or report.registry_name == 'business_fop' %}
```
Comment (PyDev777) [inline data_ocean/templates/data_ocean/emails/report.html:32]: business_fop_full? (business_fop used for short source file)

File: business_register/converter/fop_full.py [hunk 3]
```diff
@@ -290,3 +298,80 @@ def save_to_db(self, records):
...
+            new_total_records = Fop.objects.count()
+            self.update_register_field(settings.FOP_REGISTER_LIST, 'total_records', new_total_records)
+            logger.info(f'{self.reg_name}: Update total records finished successfully.')
+
+            self.measure_changes('business_register', 'Fop')
+            logger.info(f'{self.reg_name}: Report created successfully.')
+
+            logger.info(f'{self.reg_name}: Update finished successfully.')
+        self.report.update_finish = timezone.now()
+        self.report.save()
```
Comment (PyDev777) [inline business_register/converter/fop_full.py:377]: In other converters "self.report.save()" work before vacuum analyze & endpoints_cache_warm_up. Fix in this or in other converters?

**Review by PyDev777 — CHANGES_REQUESTED** [review]

File: business_register/converter/fop_full.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/converter/fop_full.py:305]: But the register is one for all type of source files.

File: data_ocean/templates/data_ocean/emails/report.html (hunk 2 above)
Comment (Iva-khar) [reply data_ocean/templates/data_ocean/emails/report.html:32]: The same.

**Pull request merged**